bluemira/power_cycle/timeline.py

```diff
@@ -0,0 +1,306 @@
+"""
+Classes to define the timeline for Power Cycle simulations.
+"""
+
+# Import
+from typing import Union  # , List
+
+from bluemira.power_cycle.base import PowerCycleABC as imported_abc
```

Comment:
dont rename imports like this. It makes everything harder to follow

Reply:
I usually do this when my classes are still frequently changing name, while I still haven't decided on a particular nomenclature for the architecture. I did not intend to leave them, but good that you mention it: I will already remove it in the new commit.